nengo/spa/state.py

```diff
@@ -0,0 +1,76 @@
+import numpy as np
+
+import nengo
+from nengo.spa.module import Module
+from nengo.networks.ensemblearray import EnsembleArray
+
+# Is the feedback parameter a big mistake or should I change the tests?
+
+class State(Module):
+    """A module capable of representing a single vector, with optional memory.
+
+    This is a minimal SPA module, useful for passing data along (for example,
+    visual input).
+
+    Parameters
+    ----------
+    dimensions : int
+        Number of dimensions for the vector
+    subdimensions : int
+        Size of the individual ensembles making up the vector.  Must divide
+        evenly into dimensions
+    neurons_per_dimensions : int
+        Number of neurons in an ensemble will be this*subdimensions
+    vocab : Vocabulary, optional
+        The vocabulary to use to interpret this vector
+    direct : boolean
+        Whether or not to use direct mode for the neurons
+    """
+
+    def __init__(self, dimensions, subdimensions=16, neurons_per_dimension=50,
+                 feedback=1.0, tau=0.1,
```

Comment:
@tcstewar please ignore the messiness of this commit and direct your attention to this file. What do you think of the `feedback` parameter? Is it being used appropriately? Previously, in `spa.Memory` the "feedback" parameter was scaled based on tau and the synapse. Why was that and should it be done here?